sirepo/pkcli/setup_dev.py

```diff
@@ -37,25 +37,29 @@ def _proprietary_codes():
     import sirepo.feature_config
     import sirepo.sim_data
     import sirepo.srdb
+    import subprocess
     import urllib.error
     import urllib.request
 
     for s in sirepo.feature_config.proprietary_sim_types():
         f = sirepo.sim_data.get_class(s).proprietary_code_tarball()
         if not f:
             continue
-        r = pkio.mkdir_parent(
+        d = pkio.mkdir_parent(
             sirepo.srdb.proprietary_code_dir(s),
-        ).join(f)
+        )
+        z = d.join(f)
         # POSIT: download/installers/flash-tarball/radiasoft-download.sh
         u = f"{cfg.proprietary_code_uri}/{s}-dev.tar.gz"
         try:
-            urllib.request.urlretrieve(u, r)
+            urllib.request.urlretrieve(u, z)
         except urllib.error.URLError as e:
             if not isinstance(e.reason, FileNotFoundError):
                 raise
-            pkdlog("uri={} not found; mocking empty file={}", u, r)
+            pkdlog("uri={} not found; mocking empty file={}", u, z)
+            t = d.join("foo.txt")
```

Comment:
I might call this README instead of foo. That way someone might be prompted to read the file and see our message.

Reply:
[2837f23](https://github.com/radiasoft/sirepo/pull/7089/commits/2837f23b92744ad1c7a55d4f2a0d7086d48c8f20)